include/mbedtls/ssl.h

```diff
@@ -553,22 +569,20 @@ typedef int mbedtls_ssl_get_timer_t( void * ctx );
  *                  this function sends or enqueues a request and does
  *                  not wait for the operation to complete.
  *
- *                  The parameters \c connection_ctx and \c cert are
+ *                  The parameters \c ssl and \c cert are
```

Comment:
Should the end of the sentence be changed from "SSL configuration" to "SSL connection"?

Reply:
Yes. Actually, `cert` may even not last that long — e.g. with an SNI callback, it depends how the SNI callback manages the certificate. Isn't it allowed to free the certificate after the handshake if renegotiation is disabled?